QCoDeS/Qcodes_contrib_drivers
# Fix CI on Master branch

* Fixes to support latest mypy
* Change intersphinx links to avoid warnings
* Bump min qcodes version to prevent pip looking for an older qcodes version that supports ipywidgets 8 (which in turn depend on netcdf4 which ships sdist but not wheels for 3.7 which causes a build failure)
* Remove nitpick warnings that are unused and may trigger warnings
* Stop using qcodes requirements file. This causes install issues when qcodes has already bumped something on master that the release does not yet support (such as ipywidgets)


Note for now I have only added mypy to the local requirements (constraints) file. We should perhaps add more important dependencies that we know often breaks on upgrade (sphinx ..) and enable dependabot here too
